build/common/installer/scripts/tomlparser-agent-config.rb

```diff
@@ -404,6 +432,24 @@ def populateSettingValuesFromConfigMap(parsedConfig)
     file.write("export MONITORING_MAX_EVENT_RATE=#{@mdsdMonitoringMaxEventRate}\n")
   end
 
+  if @mdsdUploadMaxSizeInMB > 0
+    file.write("export MDSD_ODS_UPLOAD_CHUNKING_SIZE_IN_MB=#{@mdsdUploadMaxSizeInMB}\n")
+  end
+
+  if @mdsdUploadFrequencyInSeconds > 0
+    file.write("export AMA_MAX_PUBLISH_LATENCY=#{@mdsdUploadFrequencyInSeconds}\n")
```

Comment:
why are the variables named differently here? why not MDSD_* instead of AMA_*

Reply:
> why are the variables named differently here? why not MDSD_* instead of AMA_*

This is how they named. :)